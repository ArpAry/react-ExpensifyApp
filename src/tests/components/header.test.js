import React from 'react';
import Header  from '../../js-component/ExpensifyHeader';
import toJson from 'enzyme-to-json';
import { shallow } from "enzyme";


// Enzyme.configure({
//     adapter: new Adapter()
//   });
//configure({adapter: new Adapter()});
test('should render header correctly',()=>{
    const wrapper =shallow(<Header/>);
      expect(toJson(wrapper)).toMatchSnapshot();
}); 