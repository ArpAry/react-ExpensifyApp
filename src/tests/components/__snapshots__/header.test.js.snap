// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <p>
     
    <[object Object]
      className={[Function]}
      to="/"
    >
      Home
    </[object Object]>
     
  </p>
  <p>
     
    <[object Object]
      className={[Function]}
      to="/create"
    >
      Add
    </[object Object]>
     
  </p>
  <p>
     
    <[object Object]
      className={[Function]}
      to="/help"
    >
      Help
    </[object Object]>
     
  </p>
</header>
`;
